app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ReservationsController < ApplicationController
+  before_action :set_user
+  before_action :set_reservation, only: %i[show update destroy]
```

Comment:
On line three be sure to remove update from the before action as we do not have an update method in this controller. 🤓